src/main.ts

```diff
@@ -108,6 +111,7 @@ async function findAllRepos(
     })
 
     console.log(`Found [${organization}] as orgname parameter`)
+
```

Comment:
Let's not add extra empty lines, please remove

Reply:
removed extra lines